INSTALL.rst

```diff
@@ -29,6 +29,16 @@ or clone the repository and install from local via::
 where ``-e`` is short for ``--editable`` and links the installed package to
 the current location, such that changes there take immediate effect.
 
+Additional dependencies for running the examples
+------------------------------------------------
+
+The notebooks come with additional dependencies. Information on the
+installation of the ODE-simulator AMICI is given in its
+`installation guide <http://sbml.org/Special/Software/libSBML/docs/formatted/python-api/libsbml-math.html>`_.
```

Comment:
```suggestion
`installation guide <https://github.com/AMICI-dev/AMICI/blob/master/INSTALL.md>`_.
```

Reply:
Alternative link: https://amici.readthedocs.io/en/latest/python_installation.html

Reply:
good catch, thank you!